ext/rotoscope/rotoscope.c

```diff
@@ -126,20 +89,41 @@ VALUE escape_csv_string(VALUE string) {
   return rb_funcall(string, id_gsub, 2, str_quote, str_escaped_quote);
 }
 
-static void log_trace_event_with_caller(VALUE output_buffer, VALUE io,
-                                        rs_stack_frame_t *stack_frame,
-                                        rs_stack_frame_t *caller_frame) {
-  VALUE escaped_method_name = escape_csv_string(stack_frame->tp.method_name);
-  VALUE escaped_caller_method_name =
-      escape_csv_string(caller_frame->tp.method_name);
+typedef struct {
```

Comment:
Why don't we put this inside a header file?

Reply:
It isn't being shared with another *.c file, so it didn't seem necessary.  Actually, I'm not sure why we have the rotoscope.h header file in the first place, since it is only included by rotoscope.c.